contracts/KyberProxy.sol

```diff
@@ -0,0 +1,122 @@
+pragma solidity ^0.4.24;
+
+import "./vendors/kyber/KyberNetworkProxy.sol";
+import "./vendors/kyber/ERC20Interface.sol";
+import "./interfaces/TokenConverter.sol";
+import "./utils/Ownable.sol";
+
+contract KyberProxy is TokenConverter, Ownable {
+      
+    mapping(address => mapping(address => KyberNetworkProxy)) public converterOf;
+    Token ethToken;
+
+    event Swap(address indexed sender, ERC20 srcToken, ERC20 destToken, uint amount);
+
+    constructor(Token _ethToken) public {
+        ethToken = _ethToken;
+    }
+
+    function getReturn(
+        Token srcToken, 
+        Token destToken, 
+        uint256 srcQty
+    ) external view returns (uint256) {
+        uint256 amount;
+        KyberNetworkProxy converter = converterOf[srcToken][destToken];
+        (amount,) = converter.getExpectedRate(ERC20(srcToken), ERC20(destToken), srcQty);
```

Comment:
Can't be replaced by an inline declaration of the amount variable? 
`(uint256 amount,) = converter.getExpectedRate(ERC20(srcToken), ERC20(destToken), srcQty);`